frogger.js

```diff
@@ -3,4 +3,6 @@ class Frogger {
 
     build(){};
     render(){};
+class player {
+
```

Comment:
please add generic constructor and basic offscreen canvas properties (`this.model`, `this.ctx`)